Tagesspiegel.js

```diff
@@ -185,29 +205,31 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"date": "2017-06-27T16:20:46Z",
+				"date": "2017-06-27T18:20:46+02:00",
 				"ISSN": "1865-2263",
 				"language": "de-DE",
 				"libraryCatalog": "Tagesspiegel",
 				"publicationTitle": "Der Tagesspiegel Online",
-				"url": "http://www.tagesspiegel.de/berlin/queerspiegel/bundestagsabstimmung-ohne-fraktionszwang-ehe-fuer-alle-noch-diese-woche/19984104.html",
+				"url": "https://www.tagesspiegel.de/gesellschaft/queerspiegel/bundestagsabstimmung-ohne-fraktionszwang-ehe-fuer-alle-noch-diese-woche/19984104.html",
+				"abstractNote": "Erst gestern hat Angela Merkel ihre Position zur \"Ehe für alle\" geändert - nun soll der Bundestag wohl schon am Freitag darüber abstimmen.",
+				"section": "Gesellschaft",
 				"attachments": [
 					{
 						"title": "Snapshot",
 						"mimeType": "text/html"
 					}
 				],
 				"tags": [
-					"#btw17",
-					"Angela Merkel",
-					"Bundestagswahl 2017",
+					"Angela_Merkel",
+					"Bundestagswahl_2017",
 					"CDU",
 					"CSU",
-					"Ehe für alle",
-					"Grosse-Böhmer",
+					"Ehe_fuer_alle",
+					"Grosse_Boehmer",
 					"Queerspiegel",
-					"Renate Künast",
-					"SPD"
+					"Renate_Kuenast",
+					"SPD",
+					"_btw17"
```

Comment:
And remove tags starting with an underscore.

Reply:
Why not remove the underscore instead? Because it maybe is a SEO tag?
In the original test it was present as #btw17.

Reply:
Yeah, I would guess that it's an SEO tag. #btw17 [is a thing](https://democracy-reporting.org/ua/btw17-social-media-in-the-german-parliamentary-elections/?la), but we already have "Bundestagswahl 2017" for that item.